pages/reference/supervisor/device-metrics.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Device Metrics
+---
+
+# Device Metrics
+
+The {{ $names.company.lower }} Supervisor reports metrics about your device to the cloud. These metrics are available on your balenaCloud dashboard, on the device summary page:
+
+![Device Metrics](https://www.balena.io/blog/content/images/2020/09/device-metrics-1.png)
```

Comment:
Haven't we changed something with device metrics where the path of the storage is visible as well. Maybe a new screenshot would be better + do store the image with docs themselves. One less request to make, one less dependency.